translate/test/translate.test.js

```diff
@@ -128,7 +160,7 @@ describe('translate:translate', function () {
 
       assert(sample.mocks.translate.translate.calledOnce, 'method called once');
       assert.equal(sample.mocks.translate.translate.firstCall.args.length, 3, 'method received 3 arguments');
-      assert.equal(sample.mocks.translate.translate.firstCall.args[0], text, 'method received correct first argument');
+      assert.deepEqual(sample.mocks.translate.translate.firstCall.args[0], [text], 'method received correct first argument');
```

Comment:
I would go ahead and upgrade this to the newer assertion style, but I see the merit in using a separate PR for that. Your call either way.